src/api/renderer.h

```diff
@@ -205,7 +205,6 @@ class TESS_API TessPDFRenderer : public TessResultRenderer {
   GenericVector<long int> pages_;    // object number for every /Page object
   std::string datadir_;              // where to find the custom font
   bool textonly_;                    // skip images if set
-  int jpg_quality_;                  // set JPEG quality
```

Comment:
Travis fails becaue of this change.

Reply:
https://github.com/tesseract-ocr/tesseract/blob/cbd09de7feca46543289d01b2d8482312ced2dd7/src/api/pdfrenderer.cpp#L188